modules/ROOT/pages/provisioning-vmware.adoc

```diff
@@ -23,7 +23,7 @@ mkdir ova-templates
 coreos-installer download -s "${STREAM}" -p vmware -f ova -C ./ova-templates/
 ----
 
-Alternatively, OVA images can be manually downloaded from the https://getfedora.org/coreos/download/[download page].
+Alternatively, OVA images can be manually downloaded from the https://getfedora.org/coreos/download?tab=metal_virtualized[download page]
```

Comment:
Trailing period got dropped.

Commit message nit: maybe something like `provisioning-vmware: point to correct tab on download page`?